pandas/io/stata.py

```diff
@@ -160,15 +160,29 @@
 
 Examples
 --------
+
+Creating a dummy stata for this example
+>>>df = pd.DataFrame({'animal': ['falcon', 'parrot', 'falcon',
+...                              'parrot'],
+...                   'speed': [350, 18, 361, 15]})
+>>>df.to_stata('animals.dta')
 Read a Stata dta file:
 
->>> df = pd.read_stata('filename.dta')
+>>> df = pd.read_stata('animals.dta')
 
 Read a Stata dta file in 10,000 line chunks:
+>>>values = np.random.randint(0,10, size=(20_000,1), dtype="uint8")
+>>>df = pd.DataFrame(values, columns=["i"])
+>>>df.to_stata('filename.dta')
 
 >>> itr = pd.read_stata('filename.dta', chunksize=10000)
 >>> for chunk in itr:
-...     do_something(chunk)
+...         # Operate on a single chunk
+...chunk.mean()
```

Comment:
Indentation is wrong here.  Need to align with 1 4 space indent from the `for chunk...` line.